### PR TITLE
Continually test documentation

### PR DESCRIPTION
**Pull request by ddn0**

### CODE REVIEW

File: .github/workflows/setup_ubuntu.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-apt install -yq ccache curl gcc-9 g++-9 libopenmpi-dev
+apt install -yq ccache curl doxygen gcc-9 g++-9 libopenmpi-dev
```
Comment (witchel) [inline .github/workflows/setup_ubuntu.sh:2]: My suggestion would be  doxygen doxygen-doc graphviz graphviz-doc

Comment (ddn0) [reply .github/workflows/setup_ubuntu.sh:2]: graphviz makes sense. What's the purpose of the `-doc` packages?

Comment (witchel) [reply .github/workflows/setup_ubuntu.sh]: Yeah, I thought you would ask about that.  For ci they would be pointless.  But for a developer... I dunno.  Isn't, like, being an autodidact a good thing?

Comment (witchel) [reply .github/workflows/setup_ubuntu.sh]: Oh, sorry, I thought this was `setup_dev_ubuntu.sh`.  That one should probably be updated too.

Comment (ddn0) [reply .github/workflows/setup_ubuntu.sh]: Ah sure. Updated setup_dev_ubuntu.sh.

**Pull request merged**